go/staking/api/api.go

```diff
@@ -3,7 +3,8 @@ package api
 
 import (
 	"context"
-	"errors"
+
+	"github.com/pkg/errors"
```

Comment:
Why this change?

Reply:
oh I was going to add some context to some errors, but then I refactored that stuff into other files. I'll revert this for now